src/pages/index.js

```diff
@@ -80,6 +59,61 @@ const previewExitButton = previewModal.querySelector(".modal__exit-button");
 const previewImage = previewModal.querySelector(".modal__image");
 const previewCaption = previewModal.querySelector(".modal__caption");
 
+// avatar elements
+const avatarOpenButton = document.querySelector(".profile__avatar-button");
+const avatarModal = document.querySelector("#avatar-modal");
+const avatarCloseButton = avatarModal.querySelector(".modal__exit-button");
+const avatarForm = document.forms["avatar-form"];
+const avatarFormLink = avatarForm.querySelector("#avatar-input-link");
+
+// delete elements
+const deleteModal = document.querySelector("#delete-modal");
+const deleteForm = document.forms["delete-form"];
+const deleteCloseButton = deleteModal.querySelector(".modal__exit-button");
+const deleteCancelButton = deleteModal.querySelector("#cancel-button");
+const deleteConfirmButton = deleteModal.querySelector("#delete-button");
+
+// close button assignment
+const closeButtons = document.querySelectorAll(".modal__close");
+closeButtons.forEach((button) => {
+  const popup = button.closest(".modal");
+  button.addEventListener("click", () => closeModal(popup));
+});
+
+// delete events
+deleteForm.addEventListener("submit", (evt) => {
+  handleFormSubmit(
+    evt,
+    () => {
+      return api
+        .deleteCard(selectedCardId)
+        .then((res) => {
+          closeModal(deleteModal);
+          selectedCard.remove();
+        })
+        .catch(console.error);
+    },
+    "Delete",
+    "Deleting..."
+  );
+});
+
+// avatar events
+avatarOpenButton.addEventListener("click", () => {
+  openModal(avatarModal);
+});
+avatarForm.addEventListener("submit", (evt) => {
+  handleFormSubmit(evt, () => {
+    return api
+      .editUserAvatar(avatarFormLink.value)
+      .then((data) => {
+        profileImage.src = data.avatar;
```

Comment:
Please, disable the save button in the avatar form in `then` because the input is empty on the next opening

```js
disableButton(evt.submitter, configSettings);
```